pkg/madmin/replication-commands.go

```diff
@@ -0,0 +1,123 @@
+/*
+ * MinIO Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package madmin
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"github.com/minio/minio/pkg/auth"
+)
+
+// BucketReplicationTarget represents the target bucket and site to be replicated to.
+type BucketReplicationTarget struct {
+	Endpoint     string            `json:"endpoint"`
+	Credentials  *auth.Credentials `json:"credentials"`
+	TargetBucket string            `json:"targetbucket"`
+	IsSSL        bool              `json:"isssl"`
+}
+
+func (t *BucketReplicationTarget) String() string {
+	return fmt.Sprintf("%s%s", t.Endpoint, t.TargetBucket)
```

Comment:
```suggestion
	return t.Endpoint + t.TargetBucket
```

Shouldn't there be a separator between them?